### PR TITLE
feature/939-create-me-query

### PR DESCRIPTION
**Pull request by bl0rch1d**
https://garage.atlassian.net/secure/RapidBoard.jspa?rapidView=36&projectKey=PEO&modal=detail&selectedIssue=PEO-939

### CODE REVIEW

File: app/graphql/types/image_type.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          null: true,
+          description: I18n.t("#{I18N_PATH}.fields.filepath")
+
+    def filepath
+      return nil unless object.blob
+
+      Rails.application.routes.url_helpers.rails_blob_url(object.blob)
+    end
+
+    delegate :id, to: :object
```
Comment (mmrshk) [inline app/graphql/types/image_type.rb]: It's useless. You can remove this.

Comment (bl0rch1d) [reply app/graphql/types/image_type.rb]: why?

Comment (mmrshk) [reply app/graphql/types/image_type.rb]: Run your specs without this. In response you will receive assigned id.

Comment (bl0rch1d) [reply app/graphql/types/image_type.rb]: oh, there is mismatch, wrong id. I mean object.blob.id

File: config/locales/graphql/queries.en.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+en:
+  graphql:
+    queries:
+      me: Get current user
```
Comment (kolyanjy) [inline config/locales/graphql/queries.en.yml]: add empty line

File: bin/bundle [hunk 3]
```diff
@@ -8,55 +8,63 @@
 # this file is here to facilitate running it.
```
Comment (kolyanjy) [inline bin/bundle]: what about exclude rubocop for this file?

Comment (bl0rch1d) [reply bin/bundle]: Already excluded

**Pull request merged**